### PR TITLE
It's a post about Firebat

### PR DESCRIPTION
**Pull request by jeffmeyers**
@dbelwood @vince @jackblandin I chose you guys at random to be the lucky winners.

### CODE REVIEW

File: _posts/2015-11-05-test-your-soa-with-firebat.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+* Flow
+* Process
+
+A service is a class that defines the contracts available for a particular
+resource. This may be a shopping cart, an identity provider, or any one of your
+services.
+
+A flow is a set of service calls. Flows need not be service-specific. They can
+span many services, but they define a specific workflow. For instance, to add
+an item to a cart you may need to verify that the product exists, that it is
```
Comment (dbelwood) [inline _posts/2015-11-05-test-your-soa-with-firebat.md]: `shopping` cart?  Also, maybe brief pre-amble about what business process you are discussing?

File: _posts/2015-11-05-test-your-soa-with-firebat.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+tests across our services, and to start catching problems before deploys.
+Thorough documentation and example are available on the
+[Github](https://github.com/trunkclub/firebat) page. We hope this helps you
+wrangle your company's SOA smoke testing, and encourage pull requests and
+feedback.
+
+_Interested in what you've read on the Trunk Club blog? Want to join the best-
+dressed tech team in Chicago? Check out our
+[Engineering Page](https://www.trunkclub.com/engineering) to see open positions
+and learn more._
```
Comment (dbelwood) [inline _posts/2015-11-05-test-your-soa-with-firebat.md:54]: Can we add a blurb about Engineer in Test as well?

File: _posts/2015-11-05-test-your-soa-with-firebat.md (hunk 1 above)
Comment (jeffmeyers) [inline _posts/2015-11-05-test-your-soa-with-firebat.md]: My eyes are bad and I can't quite tell what you're correcting, halp

File: _posts/2015-11-05-test-your-soa-with-firebat.md (hunk 2 above)
Comment (jeffmeyers) [inline _posts/2015-11-05-test-your-soa-with-firebat.md:54]: Can you write it? :laughing:

File: _posts/2015-11-05-test-your-soa-with-firebat.md (hunk 1 above)
Comment (jeffmeyers) [inline _posts/2015-11-05-test-your-soa-with-firebat.md]: Got it.

File: _posts/2015-11-05-test-your-soa-with-firebat.md (hunk 2 above)
Comment (dbelwood) [inline _posts/2015-11-05-test-your-soa-with-firebat.md:54]: P.S. Are you super interested in testing SOA architectures effectively?  Join our Engineers in Test - [Engineering Careers](https://www.trunkclub.com/careers)

**Comment by jeffmeyers** [comment]
@dbelwood addressed

**Pull request merged**